shell/src/main/java/tachyon/shell/command/RmCommand.java

```diff
@@ -29,6 +29,12 @@
  */
 public final class RmCommand extends WithWildCardPathCommand {
 
+  /**
+   * Constructs a new instance to remove the file specified by argv.
```

Comment:
argv is at a different level of abstraction from this file - we can just say "to remove the specified file"